site/content/contribute/more-info/mobile/e2e.md

```diff
@@ -0,0 +1,133 @@
+
+1. Create a test file based on the file structure aforementioned above.
+2. Include Zephyr identification (ID) and title in the test description, following the format of `it('[Zephyr_id] [title]')` or `it('[Zephyr_id]_[step] [title]')` if the test case has multiple steps. For test case "{{< newtabref href="https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin%3Acom.kanoah.test-manager__main-project-page#!/testCase/MM-T109" title="MM-T109 RN apps: User can't send the same message repeatedly" >}}", it should be:
+    ```javascript
+    describe('Messaging', () => {
+        it('MM-T109 User can\'t send the same message repeatedly', () => {
+            // Test steps and assertion here
+        }
+    }
+    ```
+4. Target an element using available {{< newtabref href="https://github.com/wix/Detox/blob/master/docs/APIRef.Matchers.md#matchers" title="matchers" >}}. For best results, it is recommended to match elements by unique identifiers using `testID`. The identifier should follow the following format to avoid duplication, `<location>.<modifier>.<element>.<identifier>`, where:
+  {{<note "NOTE:">}}
+  Not all fields are required. When assigning a `testID`, carefully inspect the actual render structure and pick up the minimum fields combination to create a unique value. Some examples include: `send.button` and `post.<post-id>`. 
+  {{</note>}}
+
+    * `location` - can be a parent component, a main section, or a UI screen.
+    * `modifier` - adds meaning to the `element`.
+    * `element` - common terms like `button`, `text_input`, `image`, and the like.
+    * `identifier` - could be unique ID of a post, channel, team or user, or a number to represent order.
+  
+5. Prefix each comment line with appropriate indicator. Each line in a multi-line comment should be prefixed accordingly. Separate and group test step comments and assertion comments for better readability.
+    - `#` indicates a test step (e.g. `// # Go to a screen`)
+    - `*` indicates an assertion (e.g. `// * Check the title`)
+6. Simulate user interaction using available {{< newtabref href="https://github.com/wix/Detox/blob/master/docs/APIRef.ActionsOnElement.md" title="actions" >}}, and verify user interface (UI) expectations using {{< newtabref href="https://github.com/wix/Detox/blob/master/docs/APIRef.Expect.md" title="expect" >}}. When using `action`, `match`, or another API specific to particular platform, verify that the equivalent logic is applied so that the API does not impact the other platform. Always run tests in both platforms.
```

Comment:
```suggestion
6. Simulate user interaction using available actions, and verify user interface (UI) expectations using `expect`. When using `action`, `match`, or another API specific to a particular platform, verify that the equivalent logic is applied so that the API does not impact other platforms. Always run tests in applicable platforms.
```